packages/strapi-connector-bookshelf/lib/mount-models.js

```diff
@@ -354,39 +354,29 @@ module.exports = async ({ models, target }, ctx, { selfFinalize = false } = {})
         }
         case 'belongsToMorph':
         case 'belongsToManyMorph': {
-          const association = definition.associations.find(
-            association => association.alias === name
-          );
-
-          const morphValues = association.related.map(id => {
-            let models = Object.values(strapi.models).filter(model => model.globalId === id);
-
-            if (models.length === 0) {
-              models = Object.values(strapi.components).filter(model => model.globalId === id);
-            }
-
-            if (models.length === 0) {
-              models = Object.keys(strapi.plugins).reduce((acc, current) => {
-                const models = Object.values(strapi.plugins[current].models).filter(
-                  model => model.globalId === id
-                );
-
-                if (acc.length === 0 && models.length > 0) {
-                  acc = models;
-                }
-
-                return acc;
-              }, []);
-            }
+          const isARelatedField = attr => {
```

Comment:
Can you move those utils into the modelsUtils for now are at least delcare them outside the main function to avoid recreating the function for each relation ?

Reply:
I'll move it outside of the main function but for readability I'm not confortable with spreading the code in different places, especially that this function is kind of specific to this piece of code

Reply:
I've put it outside of the main function but still in the file as I find it too specific for this piece of code to be in utils